src/dom-common-rules/homework-4-t2.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// --- TASK 2 --- //
+// Задача на обробники подій, роботу зі сховищами та атрибутами/вмістом:
+// - Напишіть html код який містить кнопку якраз посередині (вертикально і горизонтально) сторінки.
+// - В початковому стані - на кнопці має бути текст 'Turn off', фон сторінки має стати світлий.
+// - Після натиснення - на кнопці має бути текст 'Turn on', фон сторінки має стати темний.
+// - Під кнопкою має з'явитись текстове повідомлення 'Last turn off: `{DD-MM-YYYY HH:MM:SS}`', де `{DD-MM-YYYY HH:MM:SS}` - це формат часу для виведення
+// - Після повторного натиснення - на кнопці має бути текст 'Turn off', фон сторінки має стати світлий.
+// - Під кнопкою має з'явитись текстове повідомлення 'Last turn on: `{DD-MM-YYYY HH:MM:SS}`', де `{DD-MM-YYYY HH:MM:SS}` - це формат часу для виведення
+// - Логіку роботи реалізуйте в окремому js-файлі.
+
+function showDifferentBgByClick() {
+    const myDiv = document.querySelector('.myDiv');
+    const myBtn = document.querySelector('.myBtn');
+    const body = document.getElementsByTagName('body');
```

Comment:
body один, навіщо використовувати getElementsByTagName? getElementsByTagName поверне нам колекцію, querySelector же сам DOM елемент і не доведеться звертатися до колекції по індексу, щоб взяти з неї єдиний елемент
```suggestion
    const body = document.querySelector('body');
```

Reply:
Чомусь подумав, що `document.querySelector` не спрацює для тега body. Бо, мабуть звик, що querySelector має відношення тільки до класів. Виходить, до тегів теж можна звертатись за допомогою `document.querySelector`.